lnwallet/channel.go

```diff
@@ -1567,6 +1567,10 @@ func (lc *LightningChannel) remoteLogUpdateToPayDesc(logUpdate *channeldb.LogUpd
 	case *lnwire.UpdateFulfillHTLC:
 		ogHTLC := localUpdateLog.lookupHtlc(wireMsg.ID)
 
+		if ogHTLC.addCommitHeightLocal == 0 {
```

Comment:
While we're at it, the `fetchParent` has an incorrect `godoc` comment, it uses `getFetchParent` in the comment.

Reply:
Fixed